src/Fixer/ClassNotation/ConstTypeRequiredFixer.php

```diff
@@ -0,0 +1,385 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\ClassNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\WhitespacesAwareFixerInterface;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\FixerDefinition\VersionSpecification;
+use PhpCsFixer\FixerDefinition\VersionSpecificCodeSample;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+final class ConstTypeRequiredFixer extends AbstractFixer implements WhitespacesAwareFixerInterface
+{
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'Class constants must be typed.',
+            [
+                new VersionSpecificCodeSample(
+                    '<?php
+interface Doc
+{
+    const URL = "https://github.com/FriendsOfPHP/PHP-CS-Fixer/";
+}
+',
+                    new VersionSpecification(8_03_00),
+                ),
+            ],
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * Must run after NoUnneededControlParenthesesFixer, NoUselessConcatOperatorFixer.
+     */
+    public function getPriority(): int
+    {
+        return 0;
+    }
+
+    public function isCandidate(Tokens $tokens): bool
+    {
+        return \PHP_VERSION_ID >= 8_03_00
```

Comment:
perhaps add to php 8.3 rulesset?

Reply:
could be a migrating rule set maybe, however there is no 8.3 rule set yet, maybe best to add new sets in own PR's to keep the PR's smaller?

Reply:
I'm happy with both